src/HeadProvider.js

```diff
@@ -64,7 +72,7 @@ export default class HeadProvider extends React.Component {
   componentDidMount() {
     const ssrTags = document.head.querySelectorAll(`[data-rh=""]`);
     // `forEach` on `NodeList` is not supported in Googlebot, so use a workaround
-    Array.prototype.forEach.call(ssrTags, ssrTag => ssrTag.remove());
+    Array.prototype.forEach.call(ssrTags, ssrTag => safeRemove(ssrTag));
```

Comment:
Just inline `ssrTag.parentNode.removeChild(ssrTag)` here. There's no reason to add `parentNode` check here since we iterate after query.